### PR TITLE
[Add] TwilioAuth (1.0.1)

### PR DESCRIPTION
**Pull request by AdrianaPineda**
Podspec for TwilioAuth version 1.0.1

### CODE REVIEW

File: TwilioAuth/1.0.1/TwilioAuth.podspec [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  s.platform     = :ios, "8.1"
+  s.license      = { 
+    :type => "Commercial", 
+    :text => "Copyright 2011-2017 Twilio, Inc. All rights reserved. Use of this software is subject to the terms and conditions of the Twilio Terms of Service located at http://www.twilio.com/legal/tos"
+  }
+  s.author       = { "Twilio" => "help@twilio.com" }
+  s.source       = { :http    => "https://media.twiliocdn.com/sdk/ios/auth/releases/1.0.1/twilio-auth-ios-1.0.1.tar.bz2" }
+  s.vendored_frameworks   = "TwilioAuth.framework"
+  s.requires_arc          = true
+  s.pod_target_xcconfig = { 'SWIFT_VERSION' => '3' }
```
Comment (ceaglest) [inline TwilioAuth/1.0.1/TwilioAuth.podspec]: Interesting, is there a particular reason Swift 2.x not supported by the TwilioAuth SDK?

Comment (ceaglest) [inline TwilioAuth/1.0.1/TwilioAuth.podspec]: I see that 1.0.0 was updated for Swift 3+, I had missed the updated spec [PR](https://github.com/twilio/cocoapod-specs/pull/70).

**Comment by AdrianaPineda** [comment]
@ceaglest  yes, here is the TwilioAuth pod too :) https://cocoapods.org/pods/TwilioAuth
regarding the swift requirement I can try removing it, but I remember that for the first release I tried without that flag and it failed validation and this seemed to fix it.

**Comment by ceaglest** [comment]
@AdrianaPineda of course, I forgot that I had looked up your [pod](https://cocoapods.org/pods/TwilioAuth) before. Regarding Swift, our specs are currently validating without this line, so it was likely `requires_arc` that was causing issues. We have some Video customers that have production apps built on Swift 2.x, and are not currently able to migrate to Swift 3. Food for thought.

**Comment by AdrianaPineda** [comment]
@ceaglest ohh sure, I'll take a look at it thanks!

**Pull request merged**